inspirehep/modules/forms/static/scss/forms/form.scss

```diff
@@ -31,56 +31,66 @@
 }
 
 .ui-autocomplete-loading {
-     background: white url('/static/images/loading.gif') right top no-repeat;
+  background: white url('/static/images/loading.gif') right top no-repeat;
```

Comment:
This actually happened because of the autoformatting on my editor. I fixed most of the files but I missed this one.Should I revert it?